packages/codecs-data-structures/src/enum-helpers.ts

```diff
@@ -0,0 +1,104 @@
+/**
+ * Defines the "lookup object" of an enum.
+ *
+ * @example
+ * ```ts
+ * enum Direction { Left, Right };
+ * ```
+ */
+export type EnumLookupObject = { [key: string]: number | string };
```

Comment:
I'm not sure what this example is showing - should it be followed by an object of type `EnumLookupObject` like `{Left: 'Left', Right: 'Right'}`?

Reply:
Ahh I see, nice!